transforms/update-react-imports.js

```diff
@@ -11,7 +11,24 @@ module.exports = function(file, api, options) {
   const printOptions = options.printOptions || {};
   const root = j(file.source);
 
-  const reactImportPath = root
+  const variableAlreadyDeclared = variable => {
+    const foo =
+      root
+        .find(j.Identifier, {
+          name: variable,
+        })
+        .filter(
+          path =>
+            path.parent.value.type !== 'MemberExpression' &&
+            path.parent.value.type !== 'QualifiedTypeIdentifier' &&
+            path.parent.value.type !== 'JSXMemberExpression',
+        )
+        .size() > 0;
+    return foo;
```

Comment:
trying to avoid 'foo' as a variable name 
```suggestion
    const variableReferences =
      root
        .find(j.Identifier, {
          name: variable,
        })
        .filter(
          path =>
            path.parent.value.type !== 'MemberExpression' &&
            path.parent.value.type !== 'QualifiedTypeIdentifier' &&
            path.parent.value.type !== 'JSXMemberExpression',
        );
    return variableReferences.size() > 0;
```

Reply:
oh LOL oops I used `foo` so I could `console.log` the output and I forgot to change it >.>